includes/agentic-commerce/class-wc-stripe-agentic-commerce-tax-calculator.php

```
@@ -27,15 +27,27 @@ class WC_Stripe_Agentic_Commerce_Tax_Calculator {
+	 * @throws Exception When a SKU cannot be resolved to a product.
 	 */
 	public function extract_line_items_from_customization_hook(
 		WC_Stripe_Agentic_Customize_Checkout_Event $event
 	): array {
 		$line_items = [];
 
 		foreach ( $event->get_line_items() as $line_item ) {
-			$line_items[ $line_item->get_id() ] = $line_item->get_sku_id();
+			$product_id = wc_get_product_id_by_sku( $line_item->get_sku_id() );
+			if ( ! $product_id ) {
+				throw new Exception(
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '41', column '15'). (undefined)

(MissingImport)